apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/TransactionSummary.tsx

```diff
@@ -4,20 +4,18 @@ import {StyleSheet, TouchableOpacity, View} from 'react-native'
 
 import {Icon, Spacer, Text} from '../../../../components'
 import {AmountItem} from '../../../../components/AmountItem/AmountItem'
-import {BottomSheetModal} from '../../../../components/BottomSheetModal'
 import {useLanguage} from '../../../../i18n'
 import {useSelectedWallet} from '../../../../SelectedWallet'
 import {COLORS} from '../../../../theme'
 import {useTokenInfo} from '../../../../yoroi-wallets/hooks'
 import {asQuantity, Quantities} from '../../../../yoroi-wallets/utils'
 import {useStrings} from '../../common/strings'
 
-export const TransactionSummary = () => {
-  const [bottomSheetState, setBottomSheetSate] = React.useState<{isOpen: boolean; title: string; content?: string}>({
-    isOpen: false,
-    title: '',
-    content: '',
-  })
+export const TransactionSummary = ({
+  openInfoBottomSheet,
```

Comment:
I'd say the `BottomSheet` is leaking the implementation, it should be `openInfo`, if later we change to another modal, this shouldn't impact the params.